tests/badge.spec.ts

```diff
@@ -0,0 +1,79 @@
+import { test, expect } from '@playwright/test';
+import config from '../tailwind.config.cjs'
+
+const hexToRGB = (hex) => {
```

Comment:
I originally had this func in another file, but I was having difficulty importing it into badge.spec.ts.